client/dashboard/profile-wizard/index.js

```diff
@@ -74,11 +74,13 @@ class ProfileWizard extends Component {
 	componentDidMount() {
 		document.documentElement.classList.remove( 'wp-toolbar' );
 		document.body.classList.add( 'woocommerce-profile-wizard__body' );
+		document.body.classList.add( 'woocommerce-dashboard__body' );
```

Comment:
Minor but I suppose we could combine both classList.add and classList.remove calls.

Reply:
I think we can, but might lose support in IE11 with multiple properties for add/remove - https://caniuse.com/#feat=classlist